NaradaBotDemo/NaradaBotDemo/Source/CardCell.swift

```diff
@@ -19,97 +19,122 @@ class CardCell: UICollectionViewCell {
     let titleLabel: UILabel = {
         let label = UILabel()
         label.translatesAutoresizingMaskIntoConstraints = false
-        label.textColor = UIColor.darkGray
+        label.textColor = NaradaBotSyles.CardCellSyles.Colors.title
         label.backgroundColor = UIColor.clear
-        label.font = UIFont.systemFont(ofSize: 20)
+        label.font = NaradaBotSyles.CardCellSyles.Fonts.title
         return label
     }()
     
     let subtitleLabel: UILabel = {
         let label = UILabel()
         label.translatesAutoresizingMaskIntoConstraints = false
-        label.textColor = UIColor.darkGray
+        label.textColor = NaradaBotSyles.CardCellSyles.Colors.subtitle
         label.backgroundColor = UIColor.clear
-        label.font = UIFont.systemFont(ofSize: 18)
+        label.font = NaradaBotSyles.CardCellSyles.Fonts.subtitle
         return label
     }()
     
     let imageView: UIImageView = {
         let imageView = UIImageView()
         imageView.backgroundColor = UIColor.clear
         imageView.translatesAutoresizingMaskIntoConstraints = false
-        imageView.contentMode = UIViewContentMode.scaleAspectFit
+        imageView.contentMode = UIViewContentMode.scaleToFill
```

Comment:
why not scaleAspectFill?

Reply:
Because I want it to be like on the first picture, not like the second one.

![screen shot 2017-09-05 at 13 11 54](https://user-images.githubusercontent.com/15154495/30058837-fa030162-923c-11e7-8693-d85db2f6fcff.png)
![screen shot 2017-09-05 at 13 12 34](https://user-images.githubusercontent.com/15154495/30058836-f9fcdf26-923c-11e7-8752-b06fea3edeed.png)